main.py

```diff
@@ -194,23 +193,18 @@ def geocode_location(search_term):
      Output('current-time-display', 'children')],
     [Input('location-dropdown', 'value')]
 )
-def update_temp_and_time(selected_locations):
-    print(f"Selected location value: {selected_locations}")
-    if not selected_locations:
-        return html.Div("No location selected.", className='error'), html.Div("No location selected.",
-                                                                              className='error')
+
+def update_temp_and_time(selected_location):
+    if not selected_location:
+        return html.Div("No location selected.", className='error'), html.Div("No location selected.", className='error')
```

Comment:
The duplicated code isn't ideal, I'd be tempted to write a small inner function, like

```
def error_html(msg):
	div = html.Div(msg, className='error')
	return div, div
```